src/devtools/client/debugger/src/components/SecondaryPanes/CommandBar.js

```diff
@@ -70,28 +69,6 @@ function formatKey(action) {
 }
 
 class CommandBar extends Component {
-  componentWillUnmount() {
```

Comment:
Do we just not need this anymore or did it move somewhere else?

Reply:
This was running into the same serverside rendering problem bug earlier. I didn't feel particularly strongly about keeping step debugging shortcuts so I was quick to hit delete. I think I can bring it back and we can decide what to do with it when we finally undertake the big keyboard shortcuts discussion.